canonical-web-and-design/jaas.ai
# Remove docs and redirect to juju.is/docs

## Done
Remove docs and redirect to juju.is/docs
Add husky as a devDep as it was failing on pre-commit hook

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Check the header and footer link to the right place
- Check /how-it-works all like to the right place
- See that there are no missing /docs links in the templates
- Check that /docs redirects to juju.is/docs
- Check /docs/something redirects to jujju.is/docs/something

## Details
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1144
